cluster-autoscaler/vendor/sigs.k8s.io/cloud-provider-azure/pkg/provider/doc.go

```diff
@@ -16,4 +16,4 @@ limitations under the License.
 
 // Package provider is an implementation of CloudProvider Interface, LoadBalancer
 // and Instances for Azure.
-package provider // import "sigs.k8s.io/cloud-provider-azure/pkg/provider"
+package provider // import "sigs.k8s.io/cloud-provider-azure/pkg/provider
```

Comment:
Seems the test script is choking on this change.

Looks like it was fixed in v1.24.0: https://github.com/kubernetes-sigs/cloud-provider-azure/pull/1158

Reply:
I opened https://github.com/kubernetes/autoscaler/issues/4875 about this. The obvious solution is to carry the suspected fix in vendor, but I'd rather not do that if we can avoid it. Among other problems it may break anyone importing CA code (transitive dependency would be taken from original repo and not the fixed version in CA vendor in that case).

Let's see if Azure maintainers can help us with this one.